### ISSUE TITLE
HeaderFormatter PDU version test fails

### ISSUE THREAD

**Issue by daniestevez** [daniestevez/gr4-packet-modem#10]
This test is failing sometimes. Under gcc-14 it gives the error:
```
CircularBuffer::single_writer::PublishableOutputRange() - omitted publish call for 1 reserved samples
```
Under clang-18, a message that doesn't contain a `"packet_length"` is seemingly delivered to the block.

See https://github.com/daniestevez/gr4-packet-modem/actions/runs/9380186674/job/25826653038#step:6:21

**Comment by daniestevez** [comment]
I'm not seeing this in recent build. Maybe it's gone with some update of `gnuradio4`.

**Comment by daniestevez** [comment]
I've seen this error again with the latest main of gnuradio4.

**Comment by daniestevez** [comment]
This was caused by an incompatible port connection from `MsgPortOut` to `PortIn<Message>`. It has been fixed in aae750df52586cb695631fec05b0f98beba997a9.
